styles/Navbar.module.css

```diff
@@ -0,0 +1,93 @@
+.header{
+    border-bottom: 1px solid #E2E8F0;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+}
+
+.none {
+    display: none;
+}
+
+.bar {
+    display: block;
+    width: 35px;
+    height: 3px;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #222222;
+}
+
+.navmenu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navitem {
+    margin-left: 5rem;
+}
+
+.navlink{
+    font-size: 2rem;
+    font-weight: 400;
+    color: #475569;
```

Comment:
What is this color for? If it is for the grey color, it should be #494C4E. Also, make this a variable in the global css files so we can just call the variable instead of manually typing it out each time.